src/fsharp/symbols/Symbols.fsi

```diff
@@ -59,7 +61,8 @@ type [<Class>] public FSharpDisplayContext =
 /// The subtype of the symbol may reveal further information and can be one of FSharpEntity, FSharpUnionCase
 /// FSharpField, FSharpGenericParameter, FSharpStaticParameter, FSharpMemberOrFunctionOrValue, FSharpParameter,
 /// or FSharpActivePatternCase.
-type [<Class>] public FSharpSymbol = 
+[<Class>] 
+type public FSharpSymbol = 
```

Comment:
Wouldn't it be better to also remove these `public` keywords given the symbols would be public without them anyway?